sdk/storage/inc/common/storage_credential.hpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <string>
+
+struct TokenCredential
+{
+  std::string Token;
+};
+
+struct StorageSharedKeyCredential
+{
+  std::string AccountName;
+  std::string AccountKey;
```

Comment:
Key should not be public. The ctor should take both of these. Then a getter for account name and a setter only for account key so that the key can be changed in a running service.

Reply:
fixed